str.d.ts

```diff
@@ -0,0 +1,21 @@
+import type {Props} from './dom.d.ts';
+
+export {Raw, boolAttrs, voidElems, cls, len, /* TODO: map */} from './dom.d.ts';
```

Comment:
`len` probably has to be redefined, because the `Child` definition is different.